package.nls.json

```diff
@@ -18,6 +18,7 @@
     "azureFunctions.createFunctionAppAdvanced": "Create Function App in Azure... (Advanced)",
     "azureFunctions.createFunctionAppDetail": "For serverless, event driven apps and automation.",
     "azureFunctions.createNewProject": "Create New Project...",
+    "azureFunctions.createNewProjectWithDockerfile": "Create New Project With Dockerfile...",
```

Comment:
Maybe we should call it  `Create New Containerized Project...`?

I don't have a strong preference, but the learn document does refer to it as a containerized function app.

https://learn.microsoft.com/en-us/azure/azure-functions/functions-deploy-container?tabs=acr%2Cbash%2Cazure-cli&pivots=programming-language-javascript